tests/ignore.ledger

```diff
@@ -0,0 +1,32 @@
+; NoL2B begin
+~ Yearly
+    Expenses:Auto:Repair        $500.00
+    Assets
+; NoL2B end
+
+account Assets:Test1
+    note Test account
+    alias food
+    default
+
+; NoL2B begin
+account Assets:Test2
+    note Test account
+    alias food
+    default
+; NoL2B end
+
+2018-03-17 * Test 1
+    Assets:Test1                       10.00 EUR
+    Equity:Opening-Balance            -10.00 EUR
+
+; NoL2B begin
+2018-03-17 * Test 2
+    Assets:Test2
+    Equity:Opening-Balance            -10.00 EUR
+
+; NoL2B end
+2018-03-17 * Test 3
+    Assets:Test3                       10.00 EUR
+    Equity:Opening-Balance
+
```

Comment:
It looks like you're not testing the inline, non-block based version of the ignore here. We should test it too.